hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/HddsUtils.java

```diff
@@ -734,11 +735,22 @@ public static String processForDebug(ContainerCommandRequestProto msg) {
       return null;
     }
 
-    if (msg.hasWriteChunk() || msg.hasPutSmallFile()) {
+    if (msg.hasWriteChunk() || msg.hasPutBlock() || msg.hasPutSmallFile()) {
       final ContainerCommandRequestProto.Builder builder = msg.toBuilder();
       if (msg.hasWriteChunk()) {
-        builder.getWriteChunkBuilder().setData(REDACTED);
+        builder.getWriteChunkBuilder()
+            .setData(REDACTED)
+            .getChunkDataBuilder()
+            .clearChecksumData();
```

Comment:
there's a corner case: if the block field is set, it should also remove the chunk checksum from it.